.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "snarkVM"]
+	path = snarkVM
+	url = git@github.com:AleoHQ/snarkVM.git
```

Comment:
just asking do we need a specific version or just latests?

Reply:
latest is ok